tools/k8s-test/wait.go

```diff
@@ -0,0 +1,136 @@
+package main
+
+import (
+	"context"
+	"time"
+)
+
+func WaitForObjects(ctx context.Context, objects []Object) error {
+	for _, object := range objects {
+		switch object.Kind {
```

Comment:
is it expected another kind is send here?

Reply:
Yeah, there will be lots of kinds, but we only want to Wait for Deployment's and DeamonSet's so far (the others are purposefully ignored. I can add a comment to that effect.